misc/scripts/plugin_sanity_test.sh

```diff
@@ -0,0 +1,122 @@
+#!/bin/bash
+
+# Copyright 2017 VMware, Inc. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# To sanity test on the same VM using managed plugin
+# More information https://github.com/vmware/docker-volume-vsphere/issues/1020#issue-213470358
+
+echo "plugin_sanity_test: [INFO] Running plugin_sanity_test on the clean test setup..."
+
+# make sure sanity check is on clean slate and row count should be 2
+row_count=`docker plugin ls | wc -l`
+if [ $row_count != 2 ]; then
```

Comment:
yeah! I started with ```$row_count != 2``` and later added ```-z $pluginName``` .. I will remove the redundant one (this one).